website/content/docs/k8s/helm.mdx

```diff
@@ -169,16 +169,35 @@ Use these links to navigate to a particular top-level stanza.
         - gossip encryption key defined by `global.gossipEncryption.secretName`.
         To discover the service account name of the Consul client, run
             ```shell-session
-            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> charts/consul
+            $ helm template --show-only templates/client-serviceaccount.yaml <release-name> hashicorp/consul
+            ```
+        and check the name of `metadata.name`.
+
+      - `consulSnapshotAgentRole` ((#v-global-secretsbackend-vault-consulsnapshotagentrole)) (`string: ""`) - <EnterpriseAlert inline /> The Vault role for the Consul client snapshot agent.
+        The role must be connected to the Consul client snapshot agent's service account and
```

Comment:
We should break this up into two sentences.  Is there a way we could simplify the second sentence? The second sentence confused me.